### PR TITLE
Feature/data aggregation

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: src/vng/testsession/templates/testsession/session-log.html [hunk 1]
```diff
@@ -228,22 +227,74 @@ <h4>Assertions</h4>
...
+                                <th>Property</th>
+                                <th>Value</th>
+                            </thead>
+                            <tbody>
+                                <tr class="table-active">
+                                    <td>
+                                        Total scenario cases
+                                    </td>
+                                    <td>
+                                        {{ success }}
```
Comment (alextreme) [inline src/vng/testsession/templates/testsession/session-log.html]: I'd expect this to be a different variable (eg. success + failed + not_called)

Comment (AlviseSembenico) [reply src/vng/testsession/templates/testsession/session-log.html]: @alextreme my bad, fixed

File: src/vng/servervalidation/templates/servervalidation/server-run_detail.html [hunk 2]
```diff
@@ -29,43 +29,125 @@
...
+                                <th>Property</th>
+                                <th>Value</th>
+                            </thead>
+                            <tbody>
+                                <tr class="table-active">
+                                    <td>
+                                        Total calls
+                                    </td>
+                                    <td>
+                                        {{ results.0|add:results.1 }}
```
Comment (alextreme) [inline src/vng/servervalidation/templates/servervalidation/server-run_detail.html]: I'd prefer to make 'results' and 'assertions' dictionaries with the values. Adding via the template isn't necessary, it's much neater to do this in your view.

File: src/vng/testsession/templates/testsession/session-log.html [hunk 3]
```diff
@@ -228,22 +227,74 @@ <h4>Assertions</h4>
...
+                                <th>Property</th>
+                                <th>Value</th>
+                            </thead>
+                            <tbody>
+                                <tr class="table-active">
+                                    <td>
+                                        Total scenario cases
+                                    </td>
+                                    <td>
+                                        {{ success|add:failed|add:not_called }}
```
Comment (alextreme) [inline src/vng/testsession/templates/testsession/session-log.html]: same for these values, better to do this in your view

**Comment by AlviseSembenico** [comment]
@alextreme I updated it

**Comment by alextreme** [comment]
Please add tests. A lot is going on in here and we'd like to keep it working

**Comment by AlviseSembenico** [comment]
Tests added

**Comment by alextreme** [comment]
@AlviseSembenico can be merged, does have a conflict due to your other branches being merged though

**Pull request merged**